pkg/agent/addons/controllers/clusterclaim/clusterclaimer.go

```diff
@@ -309,14 +298,8 @@ func (c *ClusterClaimer) isROSA() (bool, error) {
 func (c *ClusterClaimer) isARO() (bool, error) {
 	_, err := c.Mapper.RESTMapping(schema.GroupKind{Group: "aro.openshift.io", Kind: "Cluster"}, "v1alpha1")
 	if err != nil {
-		if meta.IsNoMatchError(err) {
-			return false, nil
-		}
-		if strings.Contains(err.Error(), "the server could not find the requested resource") {
-			return false, nil
-		}
-		klog.Errorf("failed to mapping project:%v", err)
-		return false, err
+		klog.V(4).Infof("there is no mapping ARO cluster, %v", err)
```

Comment:
We do not need to throw error if the environment is OpenShift or ARO. I recommend to keep it as it is.

Reply:
we do not need to throw error, this part just judge the type of the current platform, so I think we have a debug log is sufficient